Feed/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+
+def Feed(request):
```

Comment:
Please follow [pep8](https://peps.python.org/pep-0008/#function-and-variable-names) for function name guidelines

Reply:
Ok I'll fix that